### ISSUE TITLE
Observation

### ISSUE THREAD

**Issue by sebest06** [vanniktech/Emoji#311]
Hi, first i like your work!, i tried to use in other project and work perfect.
I have a few observations that i think to be good for the project

This observations are for the master branch, v0.6?

if EmojiPopup use a EditText widget instead of EmojiEditText the integrat with other project will be more simple. And if you pass to builder a EmojiEditText the program will run in the actual same way. I think is a change for delink the EditText and the keyboard.

You may be not need the EmojiEditTextInterface, and should implemente input() method and backspace() method in the emojiPopup.

I do that, and i will a pull requests, can you think if you see that change good for the project. Please i like think about that with you.

**Comment by vanniktech** [comment]
I don't follow.

**Comment by premacck** [comment]
I think what he's trying to say is that currently, emojiEditText is required in the build() method's parameter of the EmojiPopup. But in newer versions of android, the default EditText and TextView can show the emojis by default. So can you provide another method with the default EditText for newer android versions?

**Comment by vanniktech** [comment]
Yeah, he wants to use any EditText/TextView. I like the idea, but I haven't had time to look at the implementation and make some other necessary changes.
